terraform/modules/apps/publishing-api/variables.tf

```diff
@@ -47,15 +47,26 @@ variable "desired_count" {
   default     = 1
 }
 
+variable "mesh_name" {
+  type = string
+}
+
 variable "internal_domain_name" {
   description = "The internal root root domain name"
   type        = string
   default     = "test.govuk-internal.digital"
 }
```

Comment:
This is unused and can be deleted.

Reply:
I have not rename this variable but use it to name the rabbitmq internal url.